src/gameObjects/player.js

```diff
@@ -1,7 +1,10 @@
 export default class Player {
   constructor(scene, x, y) {
     this.scene = scene;
+    this.playerSpeed = 2;
+    this.maxRange = 10;
     this.init(x, y);
+    this.addControls();
```

Comment:
**praise**: Good job on encapsulating control setup in a separate method. This improves code organization and readability.